### PR TITLE
Add resource info for win executable

### PR DESCRIPTION
**Pull request by soumeh01**

### CODE REVIEW

File: .github/workflows/eventlist.yml [hunk 1]
```diff
@@ -45,6 +45,13 @@ jobs:
         run: mkdir build
         working-directory: ./tools/eventlist
 
+      - name: Generate windows resources
```
Comment (JonatanAntoni) [inline .github/workflows/eventlist.yml:48]:
```suggestion
      - name: Generate version information
```

File: .github/workflows/eventlist.yml [hunk 2]
```diff
@@ -45,6 +45,13 @@ jobs:
         run: mkdir build
         working-directory: ./tools/eventlist
 
+      - name: Generate windows resources
+        if: github.event_name == 'release'
```
Comment (JonatanAntoni) [inline .github/workflows/eventlist.yml:49]:
```suggestion
```

File: .github/workflows/eventlist.yml (hunk 1 above)
Comment (soumeh01) [reply .github/workflows/eventlist.yml]: Addressed in https://github.com/ARM-software/CMSIS-View/pull/4/commits/b3dadd31d38d3910f228f20c3fe29964969575d4

File: .github/workflows/eventlist.yml (hunk 2 above)
Comment (soumeh01) [reply .github/workflows/eventlist.yml]: Addresed in https://github.com/ARM-software/CMSIS-View/pull/4/commits/b3dadd31d38d3910f228f20c3fe29964969575d4

**Review by JonatanAntoni — APPROVED** [review]

**Pull request merged**